odoo/osv/expression.py

```diff
@@ -703,6 +704,54 @@ def push_result(query, params):
                 subquery, subparams = query.select('"%s"."%s"' % (comodel._table, field.inverse_name))
                 push(('id', 'inselect', (subquery, subparams)), model, alias, internal=True)
 
+            elif field.type == 'properties':
+                assert len(path) == 2 and "." not in path[1]
+                assert operator in ('=', '!=', '>', '>=', '<', '<=', 'in', 'not in', 'like', 'ilike', 'not like', 'not ilike'), \
+                    f"Wrong search operator {operator!r}"
+                property_name = path[1]
+                check_property_name(property_name)
+
+                if isinstance(right, bool) and operator in ('=', '!='):
```

Comment:
Should you add `or right is None` ? (other field manage this case I think)

Reply:
changed